percona-xtradb-cluster-8.0-backup/lib/pxc/backup.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o errexit
+
+SST_INFO_NAME=sst_info
+CURL_RET_ERRORS_ARG='--curl-retriable-errors=7'
+
+INSECURE_ARG=""
+if [ -n "$VERIFY_TLS" ] && [[ $VERIFY_TLS == "false" ]]; then
+  INSECURE_ARG="--insecure"
+fi
+
+S3_BUCKET_PATH=${S3_BUCKET_PATH:-$PXC_SERVICE-$(date +%F-%H-%M)-xtrabackup.stream}
+
+log() {
+    { set +x; } 2>/dev/null
+    local level=$1
+    local message=$2
+    local now=$(date '+%F %H:%M:%S')
+
+    echo "${now} [${level}] ${message}"
```

Comment:
I think we should put timezone offset too. It might be better to use: `date +"%Y-%m-%dT%H:%M:%S%:z"`

Reply:
I try to use the same format as xbcloud